test/gtest/ucp/test_ucp_perf.cc

```diff
@@ -18,7 +18,26 @@ class test_ucp_perf : public ucp_test, public test_perf {
 protected:
     virtual void init() {
         test_base::init(); /* Skip entities creation in ucp_test */
+        ucs_log_push_handler(log_handler);
     }
+
+    virtual void cleanup() {
+        ucs_log_pop_handler();
+        test_base::cleanup();
+    }
+
+    static ucs_log_func_rc_t
+    log_handler(const char *file, unsigned line, const char *function,
+                ucs_log_level_t level, const char *prefix, const char *message,
+                va_list ap) {
+        // Ignore errors that transport cannot reach peer
+        if ((level == UCS_LOG_LEVEL_ERROR) && strstr(message, "transport to")) {
```

Comment:
does the text 100% match? We have such error code, maybe add this to output then parse here?

Reply:
You mean that there may be some other errors which contain this substring? I'll check

Reply:
yes, and probably in future